src/src/PluginStructs/P104_data_struct.cpp

```diff
@@ -324,7 +324,7 @@ void P104_data_struct::configureZones() {
  * Display the text with attributes for a specific zone
  *********************************************************/
 void P104_data_struct::displayOneZoneText(uint8_t                                 zone,
-                                          std::vector<P104_zone_struct>::iterator idx,
+                                          std::vector<P104_zone_struct>::iterator it,
```

Comment:
If you don't change anything in `it`, you can also use a `const_iterator`.
But personally I would expect this function parameter to be a `const P104_zone_struct&` so you dereference the iterator when calling it by calling `*it`.

By making it a (const) reference as parameter, you make the caller responsible to check if it is a valid pointer.
With an iterator as function parameter you can't even call the function with just an element of that type, you can only call it with an iterator now. Even if you try to call it from a `const_iterator` you get a compiler error with this function definition.